frame/nomination-pools/src/lib.rs

```diff
@@ -2144,8 +2144,8 @@ pub mod pallet {
 impl<T: Config> Pallet<T> {
 	/// Returns the pending rewards for the specified `member_account`.
 	///
-	/// In the case of error, `None` is returned.
-	pub fn pending_rewards(member_account: T::AccountId) -> Option<BalanceOf<T>> {
+	/// In the case of error, `None` is returned. Used by runtime API.
+	pub fn api_pending_rewards(member_account: T::AccountId) -> Option<BalanceOf<T>> {
```

Comment:
`who` is better suited as `member_account` suggests the provided account is a member of the pool, which might not be the case.